cansat-stm32f446/src/main.rs

```diff
@@ -37,12 +62,14 @@ mod app {
         delay: DelayUs<TIM3>,
         led: PA5<Output>,
         bme: BME280<I2c1<(Scl, Sda)>>,
+        controller: Controller<BlockSpi1<'static>, Clock, 4, 4>,
+        filename: String<11>,
```

Comment:
11 is a [magic number](https://clang.llvm.org/extra/clang-tidy/checks/readability/magic-numbers.html). Add a constant like MAX_FILENAME_LEN and document that fatfs doens't support longer filenames.